app/controllers/api/projects.php

```diff
@@ -1683,6 +1683,7 @@
         if ($enabled) {
             $mail = new PHPMailer(true);
             $mail->isSMTP();
+            $mail->SMTPAuth = (!empty($username) && !empty($password)) ? true : false;
```

Comment:
This can be simplified to 
```suggestion
            $mail->SMTPAuth = (!empty($username) && !empty($password));
```